docs/report/report.tex

```diff
@@ -45,6 +45,92 @@ \subsubsection{Stretch Goals}
+Sollte nicht explizit ein Lautsprecher angegeben werden, so wird der zuletzt aktive genutzt. Dieser hat bei \textit{is\_active} den Wert \textit{true}. \footnote{https://developer.spotify.com/documentation/web-api/guides/using-connect-web-api/}
+
+\subsubsection{Spotify Connect Lautsprecher speichern}
+Die Liste von verbundenen Geräten, die über einen Call an die Spotify API erhalten wird, enthält auch einen eindeutiges Feld \textit{id}, welches sich zusammen mit einem Namen speichern lässt.
+
+\subsubsection{In Cloud Anwendung die jeweilige Box auswählen}
+Bei der Ersteinrichtung der Box könnte sich die Cloud Anwendung mit der MAC Adresse oder einem vom Nutzer gesetzten Namen anmelden.
+Diese Box wäre dann der Cloud Anwendung bekannt und könnte auch in der Cloud Anwendung ausgewählt werden.
+
+\subsubsection{Boxdaten über Cloud Anwendung ändern}
+Dafür müsste eine API gebaut werden, damit der jeweilige Microcontroller der Box mit dem Cloud UI kommunizieren kann.
+
+\subsubsection{Unterstützung von Youtube Music}
+Eine Umsetzung könnte sich als umständlich erweisen, da es bisher noch keine dedizierte Youtube Music API gibt.
+\subsubsection{Unterstützung von Youtube}
+Youtube bietet die Möglichkeit, nach Videos zu suchen \footnote{https://developers.google.com/youtube/v3/}. Um diese Videos auf dem Raspberry als Musik abzuspielen würde sich `youtube-dl` zum downloaden der Videos als `.mp3` Dateien und  `omxplayer` zum Abspielen anbieten. Hierfür wäre allerdings ein entsprechender Lautsprecher am Raspberry erforderlich. Ein vergleichbares System zu Spotify Connect existiert derzeit in dieser Form noch nicht für Youtube.
+\subsubsection{Unterstützung von Apple Music}
+Apple Music bietet hier mit deren MusicKit JS\footnote{https://developer.apple.com/documentation/musickitjs/} eine Möglichkeit, um Musik abzuspielen.
+\subsubsection{Unterstützung von Deezer}
+Deezer lässt sich vergleichbar zu Spotify über eine API steuern. \footnote{http://developers.deezer.com/login?redirect=/api}
+\subsubsection{Unterstützung von eigener Musik (USB Stick, MicroSD Karte, Cloud)}
+Da hier extrem viele Möglichkeiten mit verschiedensten Problemen existieren, wird dieser Punkt vorest vernachlässigt.
+
+\subsection{Sonstiges}
+\subsubsection{3D Print version}
+Da unsere Box nicht übermäßig groß sein soll, müssten handelsübliche 3D-Drucker von der Größe ausreichend sein. Das Modellieren einer 3D-Print Version ist am Ende von der Expertise der Gruppe abhängig. Abgesehen davon sollte es kein besonderes Problem darstellen.
+
+\subsubsection{Sound Wiedergabe auf der Box selbst}
+Manche Pi Modelle verfügen über einen On-Board Audio Anschluss. Die Wiedergabe über diesen ist qualitativ für Musik meist ungeeignet und sollte daher über ein weiteres Audiomodul oder eine externe Soundkarte erfolgen.  Innerhalb der Raspberri Pi Reihe gibt es dafür Accessoires, die circa 20-30€ kosten. \footnote{https://www.raspberrypi.org/products/}.
```

Comment:
'Die Wiedergabe über diesen' könnte auch etwas schöner formuliert werden.

Reply:
adressed in commit ebddf4dd3e0ad75eae259bd72ee957a80ea9e62b